inspirehep/modules/theme/views.py

```diff
@@ -806,7 +806,8 @@ def login_success():
         'inspirehep_theme/login_success.html',
         user={
             'data': {
-                'email': current_user.email
+                'email': current_user.email,
+                'roles': map(lambda role: role.name, current_user.roles)
```

Comment:
minor: list comprehensions are more Pythonic `[role.name for role in current_user.roles]`.

Reply:
I always forget existence of list comprehensions 😄